io.github.sigmasd.recordapps.yml

```diff
@@ -0,0 +1,56 @@
+app-id: io.github.sigmasd.recordapps
+runtime: org.gnome.Platform
+runtime-version: "47"
+sdk: org.gnome.Sdk
+command: run.sh
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --device=all # audio + grpahics
+  - --filesystem=xdg-music # for saving recordings
+  - --share=network # local server
+modules:
+  - name: recordapps
+    buildsystem: simple
+    build-commands:
+      - DENORT_BIN=$PWD/denort ./deno compile -o record -A --include dist/ dist/webview.ts
+      - install -D record /app/bin/record
+      - install -D distro/run.sh /app/bin/run.sh
+      - chmod +x /app/bin/run.sh
+
+      - mkdir -p /app/share/applications
```

Comment:
```suggestion
```

Reply:
seems its actually needed, bot build failed because it didn't find that dir